components/odh-notebook-controller/controllers/notebook_controller_test.go

```diff
@@ -1039,3 +1053,32 @@ func createOAuthConfigmap(name, namespace string, label map[string]string, confi
 		Data: configMapData,
 	}
 }
+
+// checkCertConfigMap checks the content of a config map defined by the name and namespace
+// It triest to parse the given certFileName and checks that all certificates can be parsed there and that the number of the certificates matches what we expect.
+func checkCertConfigMap(ctx context.Context, namespace string, configMapName string, certFileName string, expNumberCerts int) {
+	configMap := &corev1.ConfigMap{}
+	key := types.NamespacedName{Namespace: namespace, Name: configMapName}
+	Expect(cli.Get(ctx, key, configMap)).Should(Succeed())
+
+	// Attempt to decode PEM encoded certificates so we are sure all are readable as expected
+	certData, _ := configMap.Data[certFileName]
```

Comment:
```suggestion
	certData := configMap.Data[certFileName]
```

that is what the linter wants you to do; fetching from a map has a single valued result and two valued result, and go will pick depending on how many params you are taking out

https://go.dev/tour/moretypes/22

Reply:
You're right, thanks, updated :slightly_smiling_face: